app/assets/stylesheets/modules/data_access.css.scss

```diff
@@ -288,7 +288,7 @@ section.data-access-next {
     border-top-left-radius: 4px;
     border-top-right-radius: 4px;
     padding: 10px;
-    width: 24%;
+    width: 230px;
```

Comment:
@jsiarto The ESI help text still spills over to 3 lines at this width for me. Any issues bumping this up to 240px?